roles/neutron-data-network/tasks/main.yml

```diff
@@ -1,12 +1,10 @@
 ---
-  #FIXME Do we need these pkgs on Rhel?
 - name: install neutron-data-network packages
   package: name={{ item }}
   with_items:
     - dnsmasq
     - dnsmasq-utils
     - ucarp
-    - radvd
```

Comment:
Nevermind, even though we don't use ipv6 Neutron still requires it for HA Routers.